authors/apps/authentication/serializers.py

```diff
@@ -182,3 +188,58 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+def email_structure(to_email, token):
+    user_info = {"user": {
+            "password": "new_password"}
+        }
+
+    # email setup
+    subject = 'Password reset'
+    message = """
+                    You're receiving this email because you invoked a password reset on
+                    Authors haven. If you think it is a mistake to receive this email just ignore it.
+                    ----- Use Postman to reset the email ----
+                        http://127.0.0.1:8000/api/user/reset-password/{}
```

Comment:
Make the base of the URL dynamic depending on the environment. For example this will not work for staging or production since the URL points to localhost.